packages/mermaid/src/manager.ts

```diff
@@ -81,7 +81,9 @@ export class MermaidManager implements IMermaidManager {
     const el = document.createElement('div');
     document.body.appendChild(el);
     try {
-      const { svg } = await _mermaid.render(id, text, el);
+      let { svg } = await _mermaid.render(id, text, el);
+      svg = svg.replace(Private.RE_VOID_ELEMENT, Private.replaceVoidElement);
+
```

Comment:
I would love to see this unit tested. For example input `'<br/>'` (which I understand we do not expect from the current version of marked but I guess user or future versions could introduce it) results in `'<br / >'` which strictly speaking is invalid XHTML. Basically:

```js
new DOMParser().parseFromString('<br>'.replace(RE_VOID_ELEMENT, replaceVoidElement), 'image/svg+xml')
```

works, but:

```js
new DOMParser().parseFromString('<br/>'.replace(RE_VOID_ELEMENT, replaceVoidElement), 'image/svg+xml')
```

fails.

Reply:
Sure, will add some coverage. 

I'm semi-disappointed we _still_ don't have any [property-based testing](https://github.com/dubzzz/fast-check) or overarching [frontend coverage](https://github.com/istanbuljs/istanbuljs) that combines `jest` and `puppeteer` outputs, but certainly won't be wading into the stack any more than i have to on this PR.